test/e2e/integration.test.ts

```diff
@@ -0,0 +1,74 @@
+import { it } from "vitest";
+import { ArkadeLightning, BoltzSwapProvider } from "../../src";
+import { SingleKey, Wallet, EsploraProvider } from "@arkade-os/sdk";
+import { exec } from "child_process";
+import { promisify } from "util";
+import { expect } from "vitest";
+
+it(
+    "should recover swept VHTLCs",
+    async () => {
+        const { arkadeLightning, wallet } = await createTestWallet();
+
+        const { stdout } = await promisify(exec)(
+            `docker exec lnd lncli --network=regtest addinvoice --amt 1000`
+        );
+        const output = stdout.trim();
+        expect(output).toBeDefined();
+        expect(output).toBeTruthy();
+        const outputJSON = JSON.parse(output);
+        expect("payment_request" in outputJSON).toBeTruthy();
+        const invoice = outputJSON.payment_request;
+
+        const swap = await arkadeLightning.createSubmarineSwap({ invoice });
+        // fund the vhtlc after fulmine is down so it can be swept
+        exec(`docker compose -f test.docker-compose.yml stop boltz-fulmine`);
+        exec(
+            `docker exec -t arkd ark send --to ${swap.response.address} --amount ${swap.response.expectedAmount} --password secret`
+        );
+
+        // generate block to expire the vhtlc
+        exec(
+            `nigiri rpc generatetoaddress 21 bcrt1qlfc0juf4csvvfkvxhzygqgh523sc9ppvq4yz3v`
+        );
+
+        // sleep 30s to let arkd sweep the vhtlc
+        await new Promise((resolve) => setTimeout(resolve, 30_000));
+        // try to refund (with boltz down and vhtlc swept)
+        await arkadeLightning.refundVHTLC(swap);
```

Comment:
would be better to verify the vtxo is swept so that the test doesnt blindly pass

Reply:
impossible it passes if vtxo is not swept because "classic offchain refund" requires boltz signature (and boltz container is down)